cg/meta/workflow/nf_analysis.py

```diff
@@ -111,6 +112,11 @@ def is_multiqc_pattern_search_exact(self) -> bool:
         If false, pattern must be present but does not need to be exact."""
         return False
 
+    @property
+    def is_gene_panel_required(self) -> bool:
+        """Return True if a gene panel is needs to be created using the information in StatusDB and exporting it from Scout."""
+        return False
```

Comment:
I'm not sure if this extra logic is needed. I'd lean towards directly calling `create_gene_panel`. If a workflow doesn't need this functionality, it can override the method in its specific API and just return None.

Reply:
I don't really like this suggestion. Overriding a method to return None would imply using a method that doesn't do what's supposed to do. In this case, calling a function named `create_gene_panel` that does not do what it's name says (does not create a gene panel) sounds super confusing to me. In my opinion that would obscure the logic of the code.

I think having an extra property is clearer and I do think this logic is needed if we want to keep a unified `config_case` method.

Reply:
Fair point, but I think that's actually the idea of having nested classes, you try to unify the logic and move it into the workflows. Probably something we should discuss and include in our coding guidelines but let's move on for now 🏁 🏁